test/modules/packages/Curl/setopt.good

```diff
@@ -0,0 +1 @@
+internal error: No Curl Support
```

Comment:
Is this a compilation failure or a runtime failure? I'd expect the .good file to have the normal output and then the .skipif file to skip the test if curl is not available.